cmd/placement/main.go

```diff
@@ -49,7 +49,7 @@ func main() {
 	}
 
 	// Start Raft cluster.
-	raftServer := raft.New(cfg.raftID, cfg.raftInMemEnabled, cfg.raftBootStrap, cfg.raftPeers, cfg.raftLogStorePath)
+	raftServer := raft.New(cfg.raftID, cfg.raftInMemEnabled, true, cfg.raftPeers, cfg.raftLogStorePath)
```

Comment:
If it is always true, then remove this argument.

Reply:
Removed. thanks.

Reply:
Send code change.